parsers/IN.py

```diff
@@ -161,6 +219,125 @@ def fetch_consumption(
     return data
```

Comment:
```suggestion
    if data is None:
        raise ParserException(
            parser="IN.py",
            message=f"{target_datetime}: No valid consumption data found for {zone_key}",
        )
    return data
```
Validate consumption will return None if the consumption is invalid.